source/include/FreeRTOSIPConfigDefaults.h

```diff
@@ -241,6 +241,11 @@
     #define ipconfigUSE_IPv6    ( 1 )
 #endif
 
+#if ( ipconfigUSE_IPv4 != 1 ) || ( ipconfigUSE_IPv6 != 1 )
```

Comment:
What is the default value for these in case one want to use it with older applications? Does it mean that library will not be backward compatible if one does not define these?